src/routes/GroupSettings/GroupSettings.connector.js

```diff
@@ -22,6 +22,7 @@ export function mapStateToProps (state, props) {
   return {
     canModerate,
     currentUser,
+    fetchPending: state.pending[FETCH_GROUP_SETTINGS],
```

Comment:
Is this related?

Reply:
So, related to a group's location?

Reply:
yes, part of why it was getting reset is the state was getting reset in the component all the time. this solved it